Configurations/50-core.conf

```diff
@@ -0,0 +1,23 @@
+# We can't make commitment to supporting Microsoft assembler,
+# because it would mean supporting all masm versions. This in
+# in turn is because masm is not really an interchangeable option,
+# while users tend to have reasons to stick with specific Visual
+# Studio versions. It's usually lesser hassle to make it work
+# with latest assembler, but tweaking for older versions had
+# proven to be daunting task. This is experimental target, for
+# production builds stick with [up-to-date version of] nasm.
+
+%targets = (
+    "VC-WIN64A-core" => {
+        inherit_from     => [ "VC-WIN64A" ],
+        cflags           => add("-DOPENSSL_SYS_WIN_CORE"),
+        lflags           => add("/NODEFAULTLIB:kernel32.lib /NODEFAULTLIB:ws2_32.lib /NODEFAULTLIB:gdi32.lib /NODEFAULTLIB:advapi32.lib /NODEFAULTLIB:crypt32.lib /NODEFAULTLIB:user32.lib"),
```

Comment:
As already mentioned /NODEFAULTLIB:kernel32.lib was sufficient for me. Can you actually confirm that all the /NODEFAULTLIBs are needed? Speaking of commentary, it would be appropriate to comment on why /NODEFAULTLIB is needed (to override references from MSCVRT.lib).

Reply:
I have built onecore version and You are right, kernel32.lib is enough